<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>511</width>
      <height>560</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="maze_tab">
     <attribute name="title">
      <string>Maze</string>
     </attribute>
     <widget class="QLabel" name="width_label">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>509</y>
        <width>40</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Width:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="width_box">
      <property name="geometry">
       <rect>
        <x>126</x>
        <y>507</y>
        <width>40</width>
        <height>21</height>
       </rect>
      </property>
      <property name="inputMask">
       <string>00</string>
      </property>
     </widget>
     <widget class="QPushButton" name="load_maze_button">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>502</y>
        <width>70</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Load </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="height_box">
      <property name="geometry">
       <rect>
        <x>222</x>
        <y>507</y>
        <width>40</width>
        <height>21</height>
       </rect>
      </property>
      <property name="inputMask">
       <string>00</string>
      </property>
     </widget>
     <widget class="QLabel" name="height_label">
      <property name="geometry">
       <rect>
        <x>174</x>
        <y>509</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Height:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="generate_button">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>503</y>
        <width>70</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
     <widget class="QPushButton" name="solve_button">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>503</y>
        <width>70</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Solve</string>
      </property>
     </widget>
     <widget class="QLabel" name="maze_label">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>0</y>
        <width>500</width>
        <height>500</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="export_button">
      <property name="geometry">
       <rect>
        <x>351</x>
        <y>503</y>
        <width>70</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Export</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="cave_tab">
     <attribute name="title">
      <string>Cave</string>
     </attribute>
     <widget class="QLabel" name="cave_label">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>0</y>
        <width>500</width>
        <height>500</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="death_label">
      <property name="geometry">
       <rect>
        <x>189</x>
        <y>508</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>D. lim.:</string>
      </property>
     </widget>
     <widget class="QLabel" name="birth_label">
      <property name="geometry">
       <rect>
        <x>96</x>
        <y>505</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>B. lim.:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="go_button">
      <property name="geometry">
       <rect>
        <x>458</x>
        <y>500</y>
        <width>40</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Go</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_edit_death">
      <property name="geometry">
       <rect>
        <x>233</x>
        <y>504</y>
        <width>40</width>
        <height>23</height>
       </rect>
      </property>
      <property name="inputMask">
       <string>0</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_edit_birth">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>503</y>
        <width>40</width>
        <height>23</height>
       </rect>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhDigitsOnly</set>
      </property>
      <property name="inputMask">
       <string>0</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="jump_spinBox">
      <property name="geometry">
       <rect>
        <x>281</x>
        <y>503</y>
        <width>42</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
     <widget class="QPushButton" name="load_cave_button">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>500</y>
        <width>80</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
     <widget class="QPushButton" name="jump_button">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>501</y>
        <width>45</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Jump</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="freq_box">
      <property name="geometry">
       <rect>
        <x>385</x>
        <y>504</y>
        <width>43</width>
        <height>22</height>
       </rect>
      </property>
      <property name="inputMask">
       <string>d990</string>
      </property>
      <property name="text">
       <string>200</string>
      </property>
     </widget>
     <widget class="QLabel" name="ms">
      <property name="geometry">
       <rect>
        <x>431</x>
        <y>508</y>
        <width>20</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ms.</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>511</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
